cms/settings/base.py

```diff
@@ -544,32 +541,43 @@
 
 
 # Email settings
-# We use SMTP to send emails. We typically use transactional email services
-# that let us use SMTP.
-# https://docs.djangoproject.com/en/2.1/topics/email/
+# https://docs.djangoproject.com/en/stable/topics/email/
+
+# Use fixed strings in case import paths move
+match env.get("EMAIL_BACKEND_NAME"):
+    case "SES":
+        EMAIL_BACKEND = "django_ses.SESBackend"
```

Comment:
```suggestion
        # django-ses uses AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY
        # See https://github.com/django-ses/django-ses
        EMAIL_BACKEND = "django_ses.SESBackend"
```

Reply:
This isn't right thought - it doesn't use those env vars. See PR description.